src/routes/Favorites/routes/Address/index.js

```diff
@@ -3,6 +3,10 @@
 import View from './FavoritesAddressViewContainer'
 
 export default {
-  path: 'address(/:address)',
+  path: 'address',
   component: View,
+  childRoutes: [{
+    path: ':address',
+    component: View,
+  }],
```

Comment:
what's wrong with optional param `path: 'address(/:address)'`?

Reply:
When user hits `.../address/0x...`, react-router fires event with final route set to `address(/:address)`. But when user hits `.../address`, react router fires the same event.

We could analyze `location.pathname` in addition to `routes`, but it looks like overkill.